bdd/spec/001_event_spec.sh

```diff
@@ -82,4 +82,43 @@ Describe "Events"
       The status should eq "0"
     End
   End
+
+  Describe "converting integers to strings when received as an event"
```

Comment:
nit `integers from global memory` since we verified the following works:
```
from @std/app import start, print, exit

event aNumber: int64

on aNumber fn(num: int64) {
  print(num.toString());
}

on start {
  const a = 5 + 5;
  emit aNumber a.getOr(0);
}
```

Reply:
oh right! fixing